test002.cpp: 函数 `f' :
test002.cpp:3: エラー: `b' は宣言されていません.
test002.cpp:3: エラー: 函数ではない `b' を呼び出しました.
